src/agent/structures/responses.ts

```diff
@@ -7,7 +7,7 @@ export interface ListChatsResponse {
   previous_page_id?: string;
   next_page_id?: string;
   chats_summary: ChatsSummary[];
-  found_chats: number;
+  total_chats: number;
```

Comment:
To be reverted - it was correct before
```suggestion
  found_chats: number;
```

Reply:
Fixed in v3.5